### ISSUE TITLE
Add support for GNU sed `~` and `+` addresses

### ISSUE THREAD

**Issue by aureliojargas** [aureliojargas/sedsed#53]
Sedsed v1.1 and earlier does not support the `~` and `+` GNU sed extensions for addresses:

```console
$ ./sedsed.py -e '1~2p'
ERROR: invalid SED command '~' at line 1
$ ./sedsed.py -e '1,~2p'
ERROR: invalid SED command '~' at line 1
$ ./sedsed.py -e '1,+2p'
ERROR: invalid SED command '+' at line 1
$
```

**Comment by aureliojargas** [comment]
Fixed by the new `sedparse` parser, see #52.

```console
$ ./sedsed.py -e '1~2p'
1~2 p
$ ./sedsed.py -e '1,~2p'
1,~2 p
$ ./sedsed.py -e '1,+2p'
1,+2 p
$ seq 10 | ./sedsed.py -d --hide=hold,patt,comm -n -e '1~2p'
1
3
5
7
9
$ seq 10 | ./sedsed.py -d --hide=hold,patt,comm -n -e '1,~2p'
1
2
$ seq 10 | ./sedsed.py -d --hide=hold,patt,comm -n -e '1,+2p'
1
2
3
$
```

**Comment by aureliojargas** [comment]
See also #33
